### ISSUE TITLE
Automatically sync assets and code in editor lobbies

### ISSUE THREAD

**Issue by DrakeFruit** [Facepunch/sbox-issues#6569]
### What can't you do?

When testing multiplayer in the editor, you have to upload your game to sbox.game for the models to sync to other clients or even the local instance. It would be much easier if the assets placed synced both on start and in real time after start like placing props in scenebox.

### How would you like it to work?

When a new instance joins, assets should be shared peer to peer or automatically uploaded to sbox.game to be downloaded on the other clients.

### What have you tried?

The only fix is to upload the game privated to sbox.game every time you add a new asset, this sucks.

**Comment by solwllms** [comment]
This should already be happening, do you have a project/repro for it not working properly?

**Comment by Nolankicks** [comment]
Yeah I just store anything I mount at runtime in a net list and mount em when the player joins. Carson said it should be happening. Probably an issue with the fake local connection networking uses.

**Comment by Nolankicks** [comment]
https://github.com/Facepunch/sbox-issues/issues/6529

**Comment by Remscar** [comment]
I get this issue with ui textures, haven't tested with other asset types.

**Comment by andy013** [comment]
Is it intended that you must upload to sbox.game in order to test networking locally?

Animgraphs are not sent to new client instances joining an editor game. Models and materials seem to download fine.

I quickly tested a decal with a jpg image and a .wav sound file and they aren't networked either.
